### PR TITLE
FUSETOOLS-2366: Fix issues in MappingViewer found by SonarQube

### PR DESCRIPTION
**Pull request by jpav**

### CODE REVIEW

File: transformation/plugins/org.jboss.tools.fuse.transformation.editor/src/main/java/org/jboss/tools/fuse/transformation/editor/internal/MappingDetailViewer.java [hunk 1]
```diff
@@ -526,10 +526,10 @@ private void managerEvent(String eventType,
                               Object oldValue,
                               Object newValue) {
         if (eventType.equals(Event.MAPPING.name())) {
-            if (equals(mapping, oldValue)) scroller.setContent(null);
+            if (mappingsEqual(mapping, oldValue)) scroller.setContent(null);
```
Comment (bfitzpat) [inline transformation/plugins/org.jboss.tools.fuse.transformation.editor/src/main/java/org/jboss/tools/fuse/transformation/editor/internal/MappingDetailViewer.java]: I suspect that Sonar will want this broken out into multiple lines

File: transformation/plugins/org.jboss.tools.fuse.transformation.editor/src/main/java/org/jboss/tools/fuse/transformation/editor/internal/MappingDetailViewer.java [hunk 2]
```diff
@@ -526,10 +526,10 @@ private void managerEvent(String eventType,
...
             else if (newValue != null) update((MappingOperation<?, ?>)newValue);
         } else if (eventType.equals(Event.MAPPING_TRANSFORMATION.name())) {
-            if (equals(mapping, oldValue)) update((MappingOperation<?, ?>)newValue);
+            if (mappingsEqual(mapping, oldValue)) update((MappingOperation<?, ?>)newValue);
```
Comment (bfitzpat) [inline transformation/plugins/org.jboss.tools.fuse.transformation.editor/src/main/java/org/jboss/tools/fuse/transformation/editor/internal/MappingDetailViewer.java]: I suspect that Sonar will want this broken out into multiple lines

File: transformation/plugins/org.jboss.tools.fuse.transformation.editor/src/main/java/org/jboss/tools/fuse/transformation/editor/internal/MappingViewer.java [hunk 3]
```diff
@@ -149,22 +149,40 @@ private void dropOnTarget() throws Exception {
         setTarget((Model)Util.draggedObject());
     }
 
-    boolean equals(MappingOperation<?, ?> mapping,
-                   Object object) {
-        if (mapping == object) return true;
-        if (mapping == null || object == null) return false;
-        if (!(object instanceof MappingOperation<?, ?>)) return false;
+    boolean mappingsEqual(MappingOperation<?, ?> mapping,
```
Comment (bfitzpat) [inline transformation/plugins/org.jboss.tools.fuse.transformation.editor/src/main/java/org/jboss/tools/fuse/transformation/editor/internal/MappingViewer.java]: Much easier to read!

**Comment by jpav** [comment]
Yes, I know, but bear with me.  I’m attacking all of these issues one class at a time to keep my sanity wrt the reports.  So, although renaming that method affected two other classes, I’ll address the rest of the Sonar stuff for those classes in separate PRs.

**Comment by bfitzpat** [comment]
Got it. +1 :)

**Pull request merged**